lib/Wikia/src/Tasks/Tasks/CreateNewWikiTask.php

```diff
@@ -97,28 +97,18 @@ public function maintenance( $server ) {
 		$output = wfShellExec( $cmd, $exitStatus );
 		$this->info( 'run updateSpecialPages.php', ['exitStatus' => $exitStatus, 'output' => $output] );
 
-		$this->info( "Remove edit lock" );
-		$variable = \WikiFactory::getVarByName( 'wgReadOnly', $wgCityId );
-		if ( isset( $variable->cv_variable_id ) ) {
-			\WikiFactory::removeVarById( $variable->cv_variable_id, $wgCityId );
-			\WikiFactory::clearCache( $wgCityId );
-		}
-
 		// SUS-3264 | set up events_local_users entries directly, instead of calling backend script
 		$this->info( "Setting up events_local_users table entries" );
 		\ListusersData::populateEventsLocalUsers( $wgCityId );
 
-		$wgMemc = wfGetMainCache();
-		$wgMemc->delete( \WikiFactory::getVarsKey( $wgCityId ) );
```

Comment:
We do not set `wgReadOnly` variable while creating a wiki so this operation is redundant.